numpy/core/numeric.py

```diff
@@ -1226,11 +1226,31 @@ def rollaxis(a, axis, start=0):
     a : ndarray
         Input array.
     axis : int
-        The axis to roll backwards.  The positions of the other axes do not
+        The axis to be rolled. The positions of the other axes do not
         change relative to one another.
     start : int, optional
-        The axis is rolled until it lies before this position.  The default,
-        0, results in a "complete" roll.
+        When ``start <= axis``, the axis is rolled back until it lies in
+        this position. When ``start > axis``, the axis is rolled until it
+        lies before this position.
+
```

Comment:
```suggestion

        The following table describes how negative values of ``start`` are
        interpreted.

```
Maybe add a bit of text here to provide context for the table that follows.